### PR TITLE
Add get_info_by_id to VOTable interface

### PR DESCRIPTION
**Pull request by mdboom**
Addresses a shortcoming shared on the mailing list thread "getting id value from xml files" by Grigoris Maravelias.  http://mail.scipy.org/pipermail/astropy/2015-March/003661.html

There is currently no way to just find an INFO element by its ID (though there is for everything else with an ID).  This adds that.

### CODE REVIEW

**Comment by pllim** [comment]
@mdboom , you already made the changes and this seems not related to Cone Search, so why am I assigned? I am confused.

**Comment by mdboom** [comment]
In astropy, we are experimenting with a new policy for PR assignment.  People assigned to a PR are responsible for reviewing and merging it -- which is usually someone different from the author of the PR.  Issue assignment still just means "I'm working on it".  So, I assigned this to you as one of the other VOTable experts on the team.

**Comment by pllim** [comment]
I can't say I am an "expert", but I can help with review. Although I cannot merge because I do not have write access.

File: astropy/io/votable/tests/data/regression.bin.tabledata.truth.1.3.xml [hunk 1]
```diff
@@ -258,7 +258,7 @@
      </TR>
     </TABLEDATA>
    </DATA>
-   <INFO ID="Error" name="Error" value="One might expect to find some INFO here, too..."/>
+   <INFO ID="ErrorInfo" name="Error" value="One might expect to find some INFO here, too..."/>
```
Comment (pllim) [inline astropy/io/votable/tests/data/regression.bin.tabledata.truth.1.3.xml]: Why did ID change from `Error` to `ErrorInfo`?

Comment (mdboom) [inline astropy/io/votable/tests/data/regression.bin.tabledata.truth.1.3.xml]: This file is what we expect to get out when we load `regression.xml`, convert it to binary and back and output it in VOTable version 1.3.  In the original file, `regression.xml`, it used to be that only `name` was specified, so `ID` was created based on it.  In order to test that `ID` and `name` are also treated separately, I added an `ID` to the original that doesn't match `name`.

**Comment by pllim** [comment]
Okay, looks fine to me. Okay to merge.

**Comment by embray** [comment]
I don't see any reason @pllim doesn't have write access.

**Pull request merged**